source/ClientApp/src/app/vendor-search/vendor-search.component.spec.ts

```diff
@@ -19,11 +19,44 @@ import { GooglePlacesService } from './Services/google-places.service';
 import { MockGoogleMapsService } from '../google-map/Services/mock-google-maps.service';
 import { MockGooglePlacesService } from './Services/mock-google-places.service';
 import { GoogleMapsService } from '../google-map/Services/google-maps.service';
+import { Subject } from 'rxjs';
+import { VendorService } from '../vendors/Services/vendor.service';
+import { VendorServices } from '../shared/models/vendor-services.model';
 
 describe('VendorSearchComponent', () => {
   let component: VendorSearchComponent;
   let fixture: ComponentFixture<VendorSearchComponent>;
   let mockVendorSearchService: VendorSearchService;
+  let mockGooglePlacesService: GooglePlacesService;
+
+  const unclaimed: VendorServices[] = [
+    {
+      vendorId: 1,
+      googleId: 'asd',
+      serviceType: 'vendor',
+      serviceName: 'Trador Joes',
+      serviceDescription: 'Place for stuff',
+      price: 1
+    },
+    {
+      vendorId: 2,
+      googleId: 'dsa',
+      serviceType: 'vendor',
+      serviceName: 'Trador Joes',
+      serviceDescription: 'Place for stuff',
+      price: 1
+    },
+  ];
+  const claimed: VendorServices[] = [
+    {
+      vendorId: 1,
+      googleId: 'asd',
+      serviceType: 'vendor',
+      serviceName: 'Trador Joes',
+      serviceDescription: 'Place for stuff',
+      price: 1
+    },
+  ];
 
```

Comment:
If this is stub data you should remove it before hand

Reply:
Its not mine so I dont want to.